styles/themes/teammember/_login-button.scss

```diff
@@ -0,0 +1,44 @@
+$login-button-font-size: 21px;
```

Comment:
$login-button-font-size - should this be converted to a standard font-size, i.e. $font-size-xlarge?